### DISCUSSION TITLE
URUWorks Website

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#92] [General]
The parent of Tero Subtitler is URUWorks, and it has a [website](https://www.uruworks.net/index.html). However, it has some issues, including that it's not mobile-friendly; and whilst GitHub is great, having a website (ideally, ad-free) is much more presentable and can help attract newcomers.

Can anyone suggest solutions? Some were suggested to me, but I don't know what the best is (if any):
- ChatGPT
- Squarespace
- Wix

**Comment by GitGianluc** [comment]
> Can anyone suggest solutions? Some were suggested to me, but I don't know what the best is (if any):
>
> - ChatGPT
> - Squarespace
> - Wix

How is ChatGPT supposed to offer a solution? Writing the website source code so that it ends up being responsive? Rather, I would be happy to have a look at the website and try to offer some solutions.

**Reply by chenlung** [reply]
My friend who suggested it: 'It's free and it creates your website.'

It's been a long time since I made a website. 😎

If not a template or automatic solution, a web designer is most likely needed.

**Reply by GitGianluc** [reply]
> If not a template or automatic solution, a web designer is most likely needed.

Let me try using a [Boostrap](https://getbootstrap.com/)-based template

**Reply by chenlung** [reply]
I don't know if I know what that is, but any improvement would be great (assuming @URUWorks approves).

By the way, are you on Discord? Would make chats easier.

**Reply by GitGianluc** [reply]
> By the way, are you on Discord? Would make chats easier.

Sorry, I’m not  😢

**Reply by chenlung** [reply]
> > By the way, are you on Discord? Would make chats easier.
> 
> 
> 
> Sorry, I’m not  😢 
> 
> 

You'd be welcome to join (unless you have a particular aversion?). Do you see the link in the main page of this GitHub? A few of us are in it already. ☺️

**Reply by URUWorks** [reply]
I approve any improvements ☺️

**Reply by GitGianluc** [reply]
OK, stay tuned for some upcoming suggestions!
